os/arch/arm/src/imxrt/imxrt_ehci.c

```diff
@@ -2666,7 +2662,7 @@ static int imxrt_qh_ioccheck(struct imxrt_qh_s *qh, uint32_t **bp, void *arg)
 		 */
 
 		**bp = qh->hw.hlp;
-		arch_flush_dcache((uintptr_t)*bp, (uintptr_t)*bp + sizeof(uint32_t));
+		arch_flush_dcache((uintptr_t) * bp, (uintptr_t) * bp + sizeof(uint32_t));
```

Comment:
As I know, we will not consider the coding rule for arch folder. But I think this is not good to read the code. Please remove the space for pointer operation.

Reply:
Done